Updated src with Bryan and My assets

Milind21 opened the pull request.

Milind21 committed:
Updated src with Bryan and My assets

Milind21 committed:
Scaled Outdoor to keep character to scale 1

blst15ki reviewed (changes requested):
- Character speed too fast, please adjust character speed to 10
- Need to add mesh colliders for the fences so the player can't pass through them
- Adjust height offset's y value under the character game object to 1

Milind21 committed:
Made requested changes

Changed speed, height offset, added mesh collider and adjustments made to the overall scene

Milind21 commented:
Added requested changes.

blst15ki commented:
Character speed feels a bit slow and the character height is lower than expected, but I will make these changes myself later.
